sim/AtlasSim.cpp

```diff
@@ -53,6 +53,16 @@ namespace atlas
         // TODO: mem usage, workload exit code
     }
 
+    void AtlasSim::setEOTMode(const std::string & eot_mode)
```

Comment:
@kathlenemagnus this function was added and "somewhat" hooked up, but does nothing at this time.